### PR TITLE
Documentation link fixes

### PR DESCRIPTION
**Pull request by 3Dgoo**
Removing defunct silverstripe-labs github link. Fixing up some broken links. Making links https if they can be https.

### CODE REVIEW

**Comment by robbieaverill** [comment]
Hey @3Dgoo - the docs site runs on major branches now. If this were merged, it would get merged up to 3 eventually, but worth noting that it will take longer than if your PR was into `3` instead of `3.6`

**Comment by 3Dgoo** [comment]
Cool. Good to know. I've changed the base branch to `3` instead of `3.6`.

**Comment by dhensby** [comment]
@robbieaverill happy?

File: docs/en/05_Contributing/01_Code.md [hunk 1]
```diff
@@ -173,15 +173,15 @@ After you have edited the file, GitHub will offer to create a pull request for y
...
-* Write [Behat integration tests](https://github.com/silverstripe-labs/silverstripe-behat-extension) for any interface changes
+* Write [Behat integration tests](https://github.com/silverstripe/silverstripe-behat-extension) for any interface changes
 * Describe specifics on how to test the effects of the patch
 * It's better to submit multiple patches with separate bits of functionality than a big patch containing lots of changes
 * Only submit a pull request for work you expect to be ready to merge. Work in progress is best discussed in an issue, or on your own repository fork.
-* Document your code inline through [PHPDoc](http://en.wikipedia.org/wiki/PHPDoc) syntax. See our 
-[API documentation](http://api.silverstripe.org/3.1/) for good examples.
-* Check and update documentation on [docs.silverstripe.org](http://docs.silverstripe.org). Check for any references to functionality deprecated or extended through your patch. Documentation changes should be included in the patch.
+* Document your code inline through [PHPDoc](https://en.wikipedia.org/wiki/PHPDoc) syntax. See our 
+[API documentation](https://api.silverstripe.org/3.5/) for good examples.
```
Comment (robbieaverill) [inline docs/en/05_Contributing/01_Code.md]: If you remove the version number from the URL then it'll be a little more future proof

Comment (3Dgoo) [inline docs/en/05_Contributing/01_Code.md]: Done. Thanks for the suggestion.

**Pull request merged**